File: C/section06/S06E06.c
#include <stdio.h>
#include <ctype.h>

int main() {
    int employeeCode, workedHours, regularWage, overtimeWage = 0;
    const int REGULAR_WAGE_PER_HOUR = 10;
    const int OVERTIME_WAGE_PER_HOUR = 20;

    printf("Please type the employee code: \n");
    scanf("%d", &employeeCode);

    printf("Please type the amount of total worked time in hours: \n");
    scanf("%d", &workedHours);

    if (workedHours > 50) {
        regularWage = REGULAR_WAGE_PER_HOUR * 50;
        overtimeWage = (workedHours - 50) * OVERTIME_WAGE_PER_HOUR;
        regularWage += overtimeWage;
    } else {
        regularWage = REGULAR_WAGE_PER_HOUR * workedHours;
    }

    printf("\nEmployee code %d\nTotal wage: R$ %d,00\nOvertime wage: R$ %d,00\n",
     employeeCode, regularWage, overtimeWage);


    return 0;
}

File: Python/section06/S06e06.py
c = int(input ("Please enter employee code: "))
n = int(input ("Please enter total worked time in hours: "))
e = 0
s = 0
regularWorkTimeLimit = 50
regularWorkTimeValue = 10
overTimeValue = 20
totalWage = 0

if (n > 50):
    e = (n - regularWorkTimeLimit) * overTimeValue
    s = regularWorkTimeLimit * regularWorkTimeValue
    totalWage = e + s
else:
    s = regularWorkTimeValue * n
    totalWage = s

print("\nTotal wage: R$ {0:.2f}\nOvertime wage: R$ {1:.2f}".format(totalWage,e))